### DISCUSSION TITLE
Prism .Net 6

### DISCUSSION THREAD

**Discussion by stevensrf11** [PrismLibrary/Prism#2729] [WPF]
Is there going to be a PRISM .Net 6 release where PRISM was compiled for .Net6 and not .Net 5. Looking over the updates and check ins on PRISM it seems since May,2022, PRISM has been dead silent.

**Comment by brianlagunas** [comment]
Don't know where you're looking, but we have been working on Prism within the past week.

![image](https://user-images.githubusercontent.com/835562/183525336-cbd51254-e56c-4566-85f5-6a7be2e189b3.png)

A Prism 6 specific assembly will ship with the next version which will include Maui support. None the less, targeting .NET 5 has zero functional impact on using it in a .NET 6 project.
